openapi/shipstation/client.bal

```diff
@@ -0,0 +1,768 @@
+# }
+# ```
+# 
+# And these headers, assuming it is 40 seconds into the current window:
+# 
+# ```
+# {
+#     "X-Rate-Limit-Limit": 60,
+#     "X-Rate-Limit-Remaining": 0,
+#     "X-Rate-Limit-Reset": 20
+# }
+# ```
+# 
+# When the limit is reached, your application should stop making requests until X-Rate-Limit-Reset seconds have elapsed. The current Rate limit for each set of the API Key and Secret is 40 requests per minute.
+# 
+# If you have any issues with the API, please email us at <apisupport@shipstation.com>
+# 
+# ## Server Responses
+# 
+# Status Code | Description
+# ------------|-------------
+# ``200`` | OK - The request was successful (some API calls may return 201 instead).
+# ``201`` | Created - The request was successful and a resource was created.
+# ``204`` | No Content - The request was successful but there is no representation to return (that is, the response is empty).
+# ``400`` | Bad Request -  The request could not be understood or was missing required parameters.
+# ``401`` | Unauthorized -  Authentication failed or user does not have permissions for the requested operation.
+# ``403`` | Forbidden - Access denied.
+# ``404`` | Not Found -  Resource was not found.
+# ``405`` | Method Not Allowed -  Requested method is not supported for the specified resource.
+# ``429`` | Too Many Requests - Exceeded ShipStation API limits. When the limit is reached, your application should stop making requests until X-Rate-Limit-Reset seconds have elapsed.
+# ``500`` | Internal Server Error - ShipStation has encountered an error.
+# 
+# ## DateTime Format and Time Zone
+# 
+# ShipStation uses the ISO 8601 combined format for dateTime stamps being submitted to and returned from the API. Please be sure to submit all dateTime values as follows:
+# 
+# yyyy-mm-dd hh:mm:ss (24 hour notation). Example - ``2016-11-29 23:59:59``
+# 
+# The time zone represented in all API responses is PST/PDT. Similarly, ShipStation asks that you make all time zone convertions and submit any dateTime requests in PST/PDT.
+@display {label: "ShipStation", iconPath: "icon.png"}
+public isolated client class Client {
+    final http:Client clientEp;
+    # Gets invoked to initialize the `connector`.
+    # The connector initialization requires setting the API credentials. 
+    # Create a [ShipStation account](https://www.shipstation.com/step1/) and obtain tokens by following [this guide](https://www.shipstation.com/docs/api/requirements/#authentication).
+    #
+    # + clientConfig - The configurations to be used when initializing the `connector` 
+    # + serviceUrl - URL of the target service 
+    # + return - An error if connector initialization failed 
+    public isolated function init(ClientConfig clientConfig, string serviceUrl = "https://ssapi.shipstation.com/") returns error? {
+        http:Client httpEp = check new (serviceUrl, clientConfig);
+        self.clientEp = httpEp;
+        return;
+    }
+    # Register Account
+    #
```

Comment:
Is there a possibility to add proper doc comments for remote functions ?

Reply:
Updated the comments